src/robot/libraries/String.py

```diff
@@ -642,6 +642,8 @@ def generate_random_string(self, length=8, chars="[LETTERS][NUMBERS]"):
         | ``[UPPER]``   | Uppercase ASCII characters from ``A`` to ``Z``. |
         | ``[LETTERS]`` | Lowercase and uppercase ASCII characters.       |
         | ``[NUMBERS]`` | Numbers from 0 to 9.                            |
+        | ``[ARABIC]``  | Arabic characters from 0x0600 to 0x0700 unicode |
+        | ``[POLISH]``  | ASCII characters and Polish diacritical signs.  |
```

Comment:
Few different things to fix/enhance:
- A note about these markers being new in RF 7.4 is needed. It probably should be at the end of the documentation where there's such a note already.
- The code uses a different range than 0x0600-0x07000 so this needs to be checked.
- Should the doc mention is the range inclusive or exclusive?
- `U+0600` could be a better to specify Unicode characters than `0x0600`.
- The correct spelling is "Unicode", not "unicode", and there should be `.` the end of a sentence.